bamboo-specs/specs/deploy-cumulus-orca-buckets.yaml

```diff
@@ -98,8 +98,8 @@ Run Ingest Integration tests:
 
 variables:
   DEPLOYMENT: sandbox
-  AWS_DEFAULT_REGION: us-west-2
-  CUMULUS_AWS_DEFAULT_REGION: us-west-2
+  #AWS_DEFAULT_REGION: us-west-2
+  #CUMULUS_AWS_DEFAULT_REGION: us-west-2
```

Comment:
I tested and this will throw an error if commented out. Recommend removing the # for now.